### ISSUE TITLE
[Bug]: sharding test report 'internal error: mpool out of space'.

### ISSUE THREAD

**Issue by Ariznawlll** [matrixorigin/matrixone#19090]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch Name

zhangxu19830126

### Commit ID

55d94f1964840ada786397ea587b27c66fbce7fd

### Other Environment Information

```Markdown
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="1043" alt="image" src="https://github.com/user-attachments/assets/93a6af5c-a3be-43c2-85fa-9c4ec3678077">

https://grafana.ci.matrixorigin.cn/explore?panes=%7B%22IiS%22:%7B%22datasource%22:%22loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bnamespace%3D%5C%22mo-shardingtest%5C%22%7D%20%7C%3D%20%60internal%20error:%20mpool%20out%20of%20space%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki%22%7D,%22editorMode%22:%22builder%22%7D%5D,%22range%22:%7B%22from%22:%221727588640000%22,%22to%22:%221727588940000%22%7D%7D%7D&schemaVersion=1&orgId=1

### Steps to Reproduce

```Markdown
测试步骤见测试文档，已发给旭哥.
```

**Comment by zhangxu19830126** [comment]
已经在其他pr解决。

**Comment by zhangxu19830126** [comment]
fixed by #19063
